### PR TITLE
Adding create Resume view

### LINKED ISSUE #95: Adding create Resume view
- Adding CreateResumeView class.
- Adding resume creation form html.
- Adding forms css.

### PR DESCRIPTION
**Pull request by AlonShakaroff**
- Adding CreateResumeView class.
- Adding resume creation form html.
- Adding forms css.
- Adding tests to the create resume views.
- Adding create new post button to side bar.

image

image

fix #95 
Dependant on #94

### CODE REVIEW

**Review by TomerNewmanPrograms — CHANGES_REQUESTED** [review]
I think you should also add test/s for the routing

File: posts/tests.py [hunk 1]
```diff
@@ -281,3 +281,18 @@ def test_delete_polls_author(self, persist_poll):
...
         with pytest.raises(expected_error):
             Poll(author=user, description=description)
+
+
+# -----------------------------------------------Post Views----------------------------------------------------------
+
+
+@pytest.mark.django_db()
+class TestPostViews:
+
```
Comment (MatanP12) [inline posts/tests.py:291]: Please remove the blank line

File: static/styles/main.css [hunk 2]
```diff
@@ -0,0 +1,170 @@
+/* ---------------------------------------------------------- GoogLe Fonts ---------------------------------------------------------- */
+@import url('https://fonts.googleapis.com/css2?family=Oleo+Script+Swash+Caps:wght@400;700&family=Poppins:ital,wght@0,300;0,500;1,600&display=swap');
+/* 
+font-family: 'Oleo Script Swash Caps', cursive;
+font-family: 'Poppins', sans-serif;
+ */
```
Comment (MatanP12) [inline static/styles/main.css:6]: Fetch with @OmerCS8 PR to remove the commented lines

File: tests/tests.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import pytest
+
+
+@pytest.mark.django_db
+class TestViews:
+
```
Comment (MatanP12) [inline tests/tests.py:6]: Please remove blank lines after class declaration

**Review by MatanP12 — CHANGES_REQUESTED** [review]

File: static/styles/main.css (hunk 2 above)
Comment (TomerNewmanPrograms) [reply static/styles/main.css:6]: yes, Alon will fetch for #89 then I will for #94 and then for #96 :)

Comment (MatanP12) [reply static/styles/main.css:6]: image

File: tests/tests.py [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import pytest
+
+
+@pytest.mark.django_db
```
Comment (yulisuliman) [inline tests/tests.py:4]: Why you need @pytest.mark.django_db ?
If you dont need then remove import pytest also.

**Review by yulisuliman — CHANGES_REQUESTED** [review]

File: posts/tests.py (hunk 1 above)
Comment (AlonShakaroff) [reply posts/tests.py:304]: 👍

File: tests/tests.py (hunk 4 above)
Comment (AlonShakaroff) [reply tests/tests.py:4]: You're right ill remove that

File: posts/templates/posts/post_detail.html [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+            <img class="rounded-circle article-img" src="{{ post.author.profile.profile_pic.url }}">
+            <a class="mr-2" href="#">{{ post.author }}</a>
+            <small class="text-muted">{{ post.date_posted|date:"F d, Y" }}</small>
+        </div>
+        <p class="article-content">{{ post.description }}</p>
+            {% if post.resume %}
+            <iframe src="{{ post.resume.resume_file.url }}" style="width: 560px;height: 780px ;border: none;"></iframe>
+            {% endif %}
+        </div>
+    </article>
```
Comment (abiton1) [inline posts/templates/posts/post_detail.html:19]:
```suggestion
    <article class="content-section">
        <div class="media-body">
            <div class="article-metadata">
                <img class="rounded-circle article-img" src="{{ post.author.profile.profile_pic.url }}">
                <a class="mr-2" href="#">{{ post.author }}</a>
                <small class="text-muted">{{ post.date_posted|date:"F d, Y" }}</small>
            </div>
            <p class="article-content">{{ post.description }}</p>
            {% if post.resume %}
                <iframe src="{{ post.resume.resume_file.url }}" style="width: 560px;height: 780px ;border: none;"></iframe>
            {% endif %}
        </div>
    </article>
```

**Review by OmerCS8 — DISMISSED** [review]
LGTM

File: posts/tests/test_posts_views.py [hunk 6]
```diff
@@ -21,3 +21,25 @@ def test_poll_detail_view(self, client, persist_poll):
         poll_detail_path = f'/post/{persist_poll.pk}/'
         poll_detail_response = client.get(poll_detail_path)
         assert poll_detail_response.status_code == 200
+
+
+@pytest.mark.django_db()
+class TestPostViews:
+    def test_post_detail_view(self, client, persist_resume, persist_poll):
```
Comment (TomerNewmanPrograms) [inline posts/tests/test_posts_views.py:28]: I think you can delete this function, we are already testing `resume_detail_response.status_code`, `poll_detail_response.status_code`

**Review by TomerNewmanPrograms — CHANGES_REQUESTED** [review]

**Review by yulisuliman — APPROVED** [review]
LGTM

**Review by MatanP12 — DISMISSED** [review]
LGTM

File: posts/tests/test_posts_views.py (hunk 6 above)
Comment (AlonShakaroff) [reply posts/tests/test_posts_views.py:28]: You are right, deleting it now 👍

**Review by TomerNewmanPrograms — APPROVED** [review]
Nice Work!

**Review by MatanP12 — APPROVED** [review]
LGTM

**Review by OmerCS8 — DISMISSED** [review]
LGTM

**Review by yulisuliman — DISMISSED** [review]

**Review by MatanP12 — DISMISSED** [review]
LGTM

**Review by TomerNewmanPrograms — APPROVED** [review]

**Review by OmerCS8 — APPROVED** [review]
LGTM

**Review by TomerNewmanPrograms — APPROVED** [review]

**Review by MatanP12 — APPROVED** [review]

**Review by yulisuliman — APPROVED** [review]

**Review by MatanP12 — APPROVED** [review]
LGTM

**Review by natifridman — APPROVED** [review]

**Review by TomerNewmanPrograms — APPROVED** [review]

**Review by yulisuliman — APPROVED** [review]

**Pull request merged**